web/tests/recent_view.test.js

```diff
@@ -232,84 +233,82 @@ people.initialize_current_user(1);
 muted_users.add_muted_user(2, 17947949);
 muted_users.add_muted_user(4, 17947949);
 
-let id = 0;
-
 const sample_messages = [];
 sample_messages[0] = {
     stream_id: stream1,
-    id: (id += 1),
+    id: 1,
```

Comment:
I changed these to numbers so it would be easier to find the messages by their ids when setting up tests, but I can change it back if that's preferred.